src/main/resources/templates/email/order-flow.ftl

```diff
@@ -37,6 +37,7 @@
         <td>${order.price!} грн</td>
     </tr>
 </table>
+<p>Перейти на KANBAN: ${url}</p>
```

Comment:
I guess, mail clients could form a link when see something starting with 'http://', but it would be more robust, I believe, if we form a link by <a> tag :)

Reply:
Okay. I didn't notice it because when I checked email it allowed me to push the link and opened app in new tab. =)